Add notes from N8CIR RSE Leaders Meeting 2025

sbland opened the pull request:
We had an in person N8CIR RSE leaders meeting in March 2025. I've consolidated the notes from the day into a list of RSE team challenges. Let me know if there is a better place to put these.

sbland committed:
Add notes from N8CIR RSE Leaders Meeting 2025

jonc125 commented on challenges/career_progression.md:
```diff
@@ -0,0 +1,21 @@
+
+Examples of career progression paths include:
+
+- ...
```
Is this intended to be a list of links to organisations' policies? Or something else?

sbland replied:
This is intended to be organisational policies. I must have missed this off in my notes so I'll follow up.

jonc125 commented on challenges/career_progression.md:
```diff
@@ -0,0 +1,21 @@
+
+The method to move between roles and grades varies between institutions.
+Some institutions now allow direct progression to higher grades. Others require
+the posting of a new job application.
```
```suggestion
Some institutions now allow direct progression to higher grades via a promotions process.
Others require people to apply for a vacant role at the higher grade, which may be open to internal applicants only.
```

jonc125 commented on challenges/index.md:
I think we've used `readme.md` in other folders.

jonc125 commented on challenges/index.md:
```diff
@@ -0,0 +1,3 @@
+# RSE Leader Challenges
+
+Challenges related to running a RSE group.
```
```suggestion
The files in this folder describe different challenges related to running a RSE group, and possible solutions.

The content was originally sourced from an N8 RSE leaders discussion, and reviewed by the wider community.
Feel free to suggest additions!
```

jonc125 commented on challenges/employment_contracts.md:
```diff
@@ -0,0 +1,11 @@
+# Employment Contracts
+
+RSE come under two types of employment contract:
```
```suggestion
RSEs in academia typically come under one of two types of employment contract:
```

jonc125 commented on challenges/employment_contracts.md:
```diff
@@ -0,0 +1,11 @@
+# Employment Contracts
```
This section doesn't really explain what the challenge is.

sbland replied:
Something along the lines of differing RSE employment contracts across institutions can make it difficult to align career path requirement and RSE expectations.

jonc125 commented on challenges/funding.md:
```diff
@@ -0,0 +1,108 @@
+# Funding
+
+Funding RSE can be challenging as we are often split between different forms of funding
```
```suggestion
Funding RSE can be challenging as we are often split between different forms of funding.
```
Why is that challenging? Or is it that different funding types bring different challenges, outlined below?

jonc125 commented on challenges/funding.md:
```diff
@@ -0,0 +1,108 @@
+
+## Glossary
+
+- **Directly Incurred (DI)**: Costs that are directly attributable to a specific
```
The UCL research services definitions may be helpful here:
* Directly Incurred costs are project-specific (i.e., they arise as a direct consequence of the project taking place), actual, and must be auditable at project level (e.g., supported by supplier invoices).
* Directly Allocated costs are not project-specific (i.e., they are incurred whether or not the project takes place), and are estimated at the project level e.g. investigator time, technician time (where not directly incurred), and estates costs.
* Indirect costs represent the costs of central and distributed services shared by other activities that are not project-specific e.g. Library services, Finance, Human Resources, and IT.

sbland replied:
Thanks @jonc125 do you have a link that I can reference here?

jonc125 replied:
https://www.ucl.ac.uk/research-innovation-services/award-services/applying-funding/costing-principles  (also talks about FEC as per below comment)

jonc125 commented on challenges/funding.md:
```diff
@@ -0,0 +1,108 @@
+  activities.
+- **Overheads**: Costs that are not directly attributable to a specific project
+  or activity.
+- **Full Economic Cost (FEC)**: The total cost of a project, including Directly
```
A bit more than this I would argue: it's a specific methodology (agreed with UK government) for calculating the true total cost of a research activity, including all the behind-the-scenes indirect/estates costs that are necessary to enable the research.

From https://www.ucl.ac.uk/research-innovation-services/award-services/applying-funding/costing-principles:

> Full Economic Costing (FEC) is a government-directed standard costing methodology used across the UK Higher Education sector for the production of consistent and transparent research project costs. The underlying principle of FEC is to establish the true cost of research, and for this to inform the amount requested from funders (the price).   
>
> In simple terms, FEC aims to capture all of the running costs of the research project, including consumables, travel costs, facility access, staff costs, estates, infrastructure costs, and any other day-to-day project costs.

jonc125 commented on challenges/funding.md:
```diff
@@ -0,0 +1,108 @@
+RSE, which is then used to calculate the cost of the RSE's time on a project. This
+rate is based on FEC and is used to recover costs from projects.
+
+**NOTE**: Applications for TRAC are currently blocked
```
What does this mean?

sbland replied:
This was mentioned at our meeting @markdturner can you please provide any more info as I think you mentioned this.

sbland replied:
Could this actually be applications for TRAC research facilities rather than TRAC?

jonc125 commented on challenges/funding.md:
```diff
@@ -0,0 +1,108 @@
+### DI funded
+
+- When charging as a DI cost, the RSE is directly funded by the project. We have
+  to be careful that there is no double counting of costs as funders may have already
```
This might need a bit more explanation. I would say double-counting is more likely in the DA case; for DI there's normally a timesheet requirement which helps avoid it.

sbland replied:
I think this refered to when an RSE is costed based on a timesheet but their time is also covered by overheads elsewhere that are claimed by the PIs overheads. Is this a possible situation?

jonc125 replied:
If they're part of a central pool or similar their salaries may be underwritten by core budget which ultimately comes (in part) from indirects on grants, and their posts may therefore have been baked into the university's calculation of its indirects rate. In such cases the institution might need to refine its calculation to allow for these staff to be charged directly to grants flexibly. However I would _hope_ that their finance systems do this reasonably transparently by just counting where salaries are charged to!

sbland replied:
I've updated with a line "This should however be managed by the institution's finance team." I think perhaps this is a page that needs further discussion and clarification at ourside now its all written down. Let me know if you are happy for this to be commited before hand or if I should remove until further discussion.

jonc125 replied:
Fine with the caveat I think.

jonc125 commented on challenges/funding.md:
```diff
@@ -0,0 +1,108 @@
+- When charging as a DI cost, the RSE is directly funded by the project. We have
+  to be careful that there is no double counting of costs as funders may have already
+  paid for the overheads indirectly.
+- Overheads generally passed to funder.
```
```suggestion
- Overheads generally paid by funder.
```

jonc125 commented on challenges/funding.md:
```diff
@@ -0,0 +1,108 @@
+
+- When charging as a DA cost, the RSE is funded by the institution and the costs
+  are shared across multiple projects.
+- Overheads generally not passed to funder
```
```suggestion
- Overheads may be paid by funder, depending on the institution's approach to costing, and the nature of the RSE's role on the project.
```

jonc125 commented on challenges/funding.md:
```diff
@@ -0,0 +1,108 @@
+
+### As a post doc
+
+Some RSEs are funded as post docs.
```
```suggestion
Some RSEs are funded as post docs, and therefore are a DI cost.
```

MarkUoLeeds replied:
Some RSEs are seen as post-docs and therefore DA cost. Members of my group specifically are added as "Researcher DA" to RGs.

jonc125 replied:
Isn't it lovely how inconsistent different institutions can be :)

In my experience, typically DA means you're either on a permanent contract with a small approximate slice of your time on a project (like an academic project lead) or part of a staff pool (like many RSEs) but not an individual on a fixed term contract (like most post docs).

MarkUoLeeds replied:
I agree hence this evidence bank challenge to consolidate (if needed?).  at least we can see what is happening. We are on "open-ended contract subject to funding" contracts. I have conversations with researchers preparing proposals and one of my roles is to ensure the 6 people in my group get funding, they do not have to do it themselves, but the quality of the work and positive outcomes often result in repeat inclusion on future work. We are a topic specialist "embedded RSE" group. Unfortunately I was unable to join the N8CIR meeting. Thanks for your reply Jonc125 and sorry to Sam for messing with the PR comments.

sbland replied:
Thanks @jonc125 and @MarkUoLeeds I think this is useful to highlight the inconsistencies. So long as we have consistent use of vocab perhaps we can at least identify the differences.

jonc125 commented on challenges/funding.md:
```diff
@@ -0,0 +1,108 @@
+## Setting a day rate
+
+Using a day rate for a project is a method of simplifying the funding application
+and cost recovery process. The day rate can be calculated using the TRAC methodology(FEC),
```
```suggestion
and cost recovery process. The day rate can be calculated using the TRAC methodology (FEC),
```

jonc125 commented on challenges/funding.md:
```diff
@@ -0,0 +1,108 @@
+
+Some considerations when setting a day rate:
+
+- The day rate may not encorporate salary inflation either due to national inflation
```
```suggestion
- The day rate may not incorporate salary inflation either due to national inflation
```

Also most UK funders won't allow you to include inflation in prices quoted.

jonc125 commented on challenges/funding.md:
```diff
@@ -0,0 +1,108 @@
+- Individual admin time
+- Time for training and professional development
+- Time for funding applications
+- Time for none project work such as supporting HPC facilities
```
```suggestion
- Time for non-project work such as supporting HPC facilities
```
I would have thought the latter was an example of institution overheads?

jonc125 commented on challenges/funding.md:
```diff
@@ -0,0 +1,108 @@
+## Funding Professional Development
+
+Funding professional development has implications on cost recovery.
+Funded is needed for the following:
```
```suggestion
Funding is needed for the following:
```

jonc125 reviewed (commented):
Thanks for submitting this! I've only read half of it so far, but thought it would be useful to share these comments and I'll get to the rest as soon as I can...

jonc125 commented on challenges/project_aquisition.md:
```diff
@@ -0,0 +1,20 @@
+- Applying directly for funding
+- Outreach events to researchers and research groups
+- Releasing reports that market skills available
+- Long term relationships with researchers and research
```
```suggestion
- Long term relationships with researchers and research groups
```
Is this what was meant?

jonc125 commented on challenges/project_aquisition.md:
```diff
@@ -0,0 +1,20 @@
+- Outreach events to researchers and research groups
+- Releasing reports that market skills available
+- Long term relationships with researchers and research
+- Via free at the point of use services (HPC, data storage, etc)
```
```suggestion
- Via free at the point of use services (support drop-ins, HPC, data storage, etc)
```

jonc125 commented on challenges/project_aquisition.md:
```diff
@@ -0,0 +1,20 @@
+## Timeframes
+
+The timeframes for project acquisition can vary depending on the method used.
+RSE teams are often not given enough lead in time when approached with a project
```
```suggestion
RSE teams are often not given enough lead-in time when approached with a project
```

jonc125 commented on challenges/project_management.md:
```diff
@@ -0,0 +1,21 @@
+# Project Management
```
I was expecting to see comments on agile methods etc. here - timesheets I'd consider more 'administration' and perhaps go under funding?

sbland replied:
We discussed timesheets in the context of a project management discussion which is why it ended up here. I will split this up into Funding and project management (where linked to time planning).

jonc125 commented on challenges/project_management.md:
```diff
@@ -0,0 +1,21 @@
+
+## Timesheets
+
+- Timesheets are required by some funders (I.e UKRI).
```
```suggestion
- Timesheets are required by some funders (e.g. UKRI, EU).
```

jonc125 commented on challenges/project_management.md:
```diff
@@ -0,0 +1,21 @@
+## Timesheets
+
+- Timesheets are required by some funders (I.e UKRI).
+- Timesheets are required by TRAC methodology.
```
Not sure this is true in general, e.g. UKRI use TRAC but timesheets are only required for DI staff not 100% on the project. Is the reference here to when costing as a TRAC **research facility** specifically?

sbland replied:
Ah I think this makes some comments in the funding around TRAC make more sense. I was mixing up comments on TRAC in general and TRAC research facilities. Would I be correct in rephrasing this as "Timesheets are required by TRAC research facilities"?

jonc125 replied:
Possibly; I've not run one so am not sure of the exact requirements. @markdturner ?

sbland replied:
I've amended to "May be required" so this can be amended in the future.

jonc125 commented on challenges/project_management.md:
```diff
@@ -0,0 +1,21 @@
+- Timesheets are required by TRAC methodology.
+- Timesheets can be used to compare expected FTE on a project with actual FTE.
+- There is an overhead to completing timesheets.
+- There can sometimes be a discrepency between actual time spent and the time
+  recorded on the timesheet to be submitted to the funder.
```
This should be avoided since it contravenes the terms of the grant agreement. Albeit minor approximation is unlikely to be an issue.

sbland replied:
I will rephrase this as "Recording actual time spent can be challenging particularly where time spent could be applied to multiple projects or activities". How does that sound?

jonc125 replied:
Sounds good!

jonc125 commented on challenges/project_management.md:
```diff
@@ -0,0 +1,21 @@
+- Toggl
+- In house bespoke
+- Excel
+
```
We use Harvest (and Forecast for planning).

jonc125 commented on challenges/recruitment.md:
```diff
@@ -0,0 +1,21 @@
+- We can split job adverts into two sections.
+- Expectation of the role. This should be very general to ensure that the RSE
+  can adapt to a variety of projects and roles.
+- Skills and competencies. This should be more specific and can be tailored to
```
I would say all parts of the JD should be general and kept consistent across hiring rounds. This helps if you're looking to implement a promotion scheme or similar. It's the job **advert** where you can be specific with extra requirements for a particular hire.

jonc125 commented on challenges/skills_and_competencies.md:
```diff
@@ -0,0 +1,51 @@
+
+- The most important skill is being able to learn and adapt.
+- More senior RSEs develop a greater breadth of skills and knowledge rather
+  than becoming technical experts in a particular area.
```
Not sure about 'rather than' - that's also a good pathway to allow!

jonc125 commented on challenges/skills_and_competencies.md:
```diff
@@ -0,0 +1,51 @@
+## Skill retention
+
+- Some skills are difficult to retain particularly in HPC support where the need can
+  vary over time.
```
As in, the level of demand for the service?

jonc125 commented on challenges/team_structure.md:
```diff
@@ -0,0 +1,41 @@
+- It can be expensive and difficult to bring in an entirely embedded RSE into the
+  central team.
+- Some RSE teams have a hard boundary between the central team and the embedded RSEs.
+- In these teams the central team is not responsible for the funding or work of
```
```suggestion
  In these teams the central team is not responsible for the funding or work of
```

jonc125 commented on challenges/team_structure.md:
```diff
@@ -0,0 +1,41 @@
+
+### Research Data Managers
+
+- How we cost these roles is more challenging as they are less likely to be directly
+  associated with a project.
```
Not necessarily! Our data stewards often are...

sbland replied:
I've rephrased this to "when not directly associated with a project". This is probably a point to review anyway as research data managers/stewards are increasingly embedded in RSE/dRTP teams.

jonc125 commented on challenges/team_structure.md:
```diff
@@ -0,0 +1,41 @@
+
+## Connection to Central IT
+
+Some RSE teams are embedded in central RSE while others exist within a faculty
```
```suggestion
Some RSE teams are embedded in central IT while others exist within an academic faculty.
Others are hybrid or uniquely situated.
```

jonc125 commented on challenges/index.md:
```diff
@@ -0,0 +1,3 @@
+# RSE Leader Challenges
```
Would links to resources like https://www.ucl.ac.uk/advanced-research-computing/collaborations-and-consultancy/ways-working be useful to describe how different teams operate?

jonc125 commented on challenges/training.md:
```diff
@@ -0,0 +1,22 @@
+# Training
+
+There are two areas of training relevent to RSE teams:
```
```suggestion
There are two areas of training relevant to RSE teams:
```

jonc125 reviewed (commented):
I've now read through the rest. Thanks Sam!

sbland committed:
Update challenges/career_progression.md

Extend note on promotion process

Co-authored-by: Jonathan Cooper <github.com@jonc.me.uk>

sbland committed:
Update challenges/index.md

Extend note on RSE leaders challenges

Co-authored-by: Jonathan Cooper <github.com@jonc.me.uk>

sbland committed:
Apply suggestions from code review

Minor typo amends from PR comments

Co-authored-by: Jonathan Cooper <github.com@jonc.me.uk>

sbland committed:
Rename index.md to README.md

sbland committed:
Updates to challenges in response to PR comments
- Updated descriptions in career progression and employment contracts
- Amends to bullet points in project aquisition
- Updates to timesheet bullet points in project management

sbland committed:
Update funding page in response to PR comments

jonc125 commented on challenges/funding.md:
```diff
@@ -85,17 +109,18 @@ be free at the point of use or we need to cost recover from projects.
 include:
 
-- Institution overheads such as estates, facilities, administration, and finance
+- None RSE institution overheads such as estates, facilities, administration, and
```
```suggestion
- Non-RSE institution overheads such as estates, facilities, administration, and
```

sbland committed:
Update challenges/funding.md

Minor typo amend

Co-authored-by: Jonathan Cooper <github.com@jonc.me.uk>

sbland committed:
Updated team structure doc
- include intro
- rephrase points on research data managers

sbland committed:
Minor amends to phrasing in skills and competencies doc
